app/js/components/orderEntry/addForm/AddForm.jsx

```diff
@@ -307,20 +307,28 @@ export class AddForm extends React.Component {
     return (
       <div>
         { this.props.drugUuid && this.renderDrugOrderForms() }
-        { this.state.draftOrders.length > 0 && this.renderDraftDataTable()}
+        {/* { this.state.draftOrders.length > 0 && this.renderDraftDataTable()} */}
```

Comment:
You could use `this.props.draftOrders.length > && this.renderDraftDataTable()`

Reply:
I commented it out because there is no more use/need for it, seen=ing as the draft table is now being rendered from elsewhere

Reply:
```SearchAndAddOrder``` component

Reply:
okay